### ISSUE TITLE
Minor typo

### ISSUE THREAD

**Issue by xSeditx** [MicrosoftDocs/cpp-docs#1653]
VCOMPD.LID should be VCOMPD.LIB I presume, if this is incorrect and it is indeed .LID ignore and close but I doubt this to be the case as immediately after it mentions VCOMPD.LIB

**Comment by corob-msft** [comment]
@xSeditx 

I assume you're talking about the [OpenMP Library Reference](https://docs.microsoft.com/cpp/parallel/openmp/reference/openmp-library-reference) article.

You could be right, but I suspect VCOMP.LIB and VCOMPD.LIB may actually be import libraries for vcomp140.dll and vcomp140d.dll. The first DLL is part of the C++ Redistributables package. There's a typo here for sure, but maybe it's a bigger typo than just one letter.

@mikeblome Do you know more about the OpenMP libraries? Those docs don't really have an owner.

**Comment by xSeditx** [comment]
Yes apologies I just clicked the issues and it brought me here. 

https://docs.microsoft.com/en-us/cpp/parallel/openmp/reference/openmp-library-reference?view=vs-2019

Is the page I noticed it on,

```
OpenMP run-time library | Characteristics
-- | --
VCOMP.LIB | Multithreaded, dynamic link (import library for VCOMP.LIB).
VCOMPD.LIB | Multithreaded, dynamic link (import library for VCOMPD.LID) (debug)
```

It would not be so bad but my initial thoughts when I seen it was that perhaps .LID was just some sort of Debug thing since it is the debug library however almost the very next line it calls it .LIB. If it is indeed a typo it is one that might lead to some confusion. 
Do I understand correctly that should say `Import Library for VCOMPD.DLL` ? If so then yes that is a bigger issue.
